source/reviews/respect.txt

```diff
@@ -0,0 +1,87 @@
+  feel free to ask why they made a particular change. Not knowing is OK, and asking
+  “Why” leaves a written record that will help answer this question in the future.
+  Sometimes, “I'm curious, why did you decide to do it that way?” can help the author
+  to rethink their decision.
+
+- Find an end. If you like things neat, it‘s tempting to go over a review over and
+  over until it’s perfect, dragging it out for longer than necessary. It‘s soul-deadening
+  for the recipient, though. Keep in mind that “LGTM” does not mean “I vouch my
+  immortal soul this good”, but “looks good to me”. If it looks good, move on.
+  That doesn’t mean you shouldn‘t be thorough. It’s a judgment call.
+  And if there are bigger refactorings to be done, move them to a new PR.
+
+- Reply within a reasonable time frame. Please don't leave the reviewee waiting
+  for a long time, keeping in mind timezones and different working hours. If you
+  cannot get to a review within ~24h, please leave a short comment on the PR
+  saying so (and when you can). And if you missed that window, please be courteous
+  if the reviewee sends you a reminder.
+
+- If you will be on vacation or otherwise OOO for more than a few days, please
+  set your status in the Slack to indicate this (e.g. adding ‘(OOO until )’) and
+  blocking your calendar.
+
+- Mention the positives. It‘s very easy to get into the mindset of “find ALL the flaws”,
+  but acknowledging the positives helps keep things civil, and brightens the recipient’s
+  day. No need to be all fake smiles, but if there's a good decision, or if somebody takes
+  on a really grungy task, acknowledging that is a nice thing to do.
+  Conversely, a “thank you” to the reviewers is occasionally a nice thing, too.
+
+Don't
++++++
+
+- Don't shame people. “How could you not see this” is a very unhelpful thing to say.
+  Assume that your colleagues do their best, but occasionally make mistakes. That's
+  why we have reviews - to spot those mistakes. While flawless PRs are awesome,
+  flawed ones are the norm.
+
+- Don't use extreme or very negative language. Please don‘t say things like
+  “no sane person would ever do this” or “this is terrible”, whether it’s about
+  the PR you‘re reviewing or about the surrounding content. While it might
+  intimidate the reviewee into doing what you want, it’s not helpful in the long run -
+  they will feel incapable, and there is not much info in there to help them improve.
+  “This is a good start, but it could use some work” or “This needs some cleanup”
+  are nicer ways of saying it. Discuss the content, not the person.
+
+- Don't discourage tool use. If people use the automated formatter, be grateful
+  they‘re willing to give up the power of formatting in favor of ensuring consistency.
+  Think carefully before enforcing your own preferences over it.
+
+- Don't bike shed. Always ask yourself if this decision really matters in the long
```

Comment:
I haven't heard "bike shed," it'd be helpful for me if the meaning were written out